painting/src/main/java/com/ataulm/artcollector/painting/ui/PaintingActivity.kt

```diff
@@ -27,12 +34,46 @@ class PaintingActivity : AppCompatActivity() {
         val paintingId = intent.data!!.pathSegments.last()
         injectDependencies(PaintingId(paintingId))
 
+        // TODO: check with app links that it still works and doesn't care about postponed transitions
```

Comment:
:x: I didn't check yet what happens when this activity is started from an app link. In this case, we don't want to cause any dodgy behaviour with the `postponeEnterTransition()` because there's no reason to postpone any transition when coming from a website.

Reply:
https://github.com/ataulm/art-collector/pull/19#issuecomment-441367212

there's no issue